### ISSUE TITLE
[Feat. Req.] Add conversation in this git + multiple language

### ISSUE THREAD

**Issue by RisedSky** [louislam/uptime-kuma-wiki#93]
Would like to add french support for this :)
Also, can you enable conversation on this repo so we can debate about some translation / content to add ? ty !

**Comment by CommanderStorm** [comment]
I don't think we could maintain a french translation of the wiki as none of the maintainers does speak french (I think).
The differerences that would inevitable come up or the people writing issues in french are not something that I really want to deal with.
I don't want to recive more issues like https://github.com/louislam/uptime-kuma/issues/3911 https://github.com/louislam/uptime-kuma/issues/4678 https://github.com/louislam/uptime-kuma/issues/4378. They are really hard to handle and trowing deepl at problems does not solve them.

If you want to create a Third party wiki in french you can do so.
We can list it under https://github.com/louislam/uptime-kuma/wiki/%F0%9F%94%A7-How-to-Install#unofficial--experimental if you are transparent about:
- that the project is not a french project
  => we can't support in french. If they write in french, they will bee put through deepl
- That that wiki is not official and might be arbitrairily different/out of date.
  It has to make it obvious that it is not a first party wiki, but something by you.
  => If it were to look official, this might lead to users being disappointed in us for possible bad/outdated advice on said wiki.

> can you enable conversation

I think you meant github discussions.
We have discussed this in the past here https://github.com/louislam/uptime-kuma/issues/92 and in the more recent issues linked to it.
Honestly, don't know what value this would add. For a mostly unmoderated (=maintainers don't get emails) experienced, [there is reddit](https://www.reddit.com/r/UptimeKuma/), for issues there are issues and for code there are PRs.

> debate about some translation

What translation is unclear? We can add screenshots to [weblate](https://weblate.kuma.pet/translate/uptime-kuma) if one of them is unclear.
(documented here https://github.com/louislam/uptime-kuma/pull/4794 and in the other translation PRs)

> debate about content to add

Why not use issues for this?
